src/components/CategoryFilter.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
+
+const CategoryFilter = ({ event }) => {
+  const categoryList = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi', 'All'];
+
+  return (
+    <div>
+      <select name="categories" onChange={event}>
+        {categoryList.map(item => <option selected={item === 'Action' ? 'selected' : null} value={item} key={`${item}_option`}>{item}</option>)}
+      </select>
+    </div>
```

Comment:
* [ ] `CategoryFilter` merges `All` select option with available categories [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/AllReturnsAllCategories.png)
**_NOTE:_** don't add `All` to the array, instead add it inside `select` before the loop.